src/Okta.Idx.Sdk/IdxContext.cs

```diff
@@ -10,11 +10,11 @@ namespace Okta.Idx.Sdk
     /// </summary>
     public class IdxContext : IIdxContext
     {
-        private readonly string _codeVerifier;
-        private readonly string _codeChallenge;
-        private readonly string _codeChallengeMethod;
-        private readonly string _interactionHandle;
-        private readonly string _state;
+        private string _codeVerifier;
```

Comment:
I think this object should remain immutable, in what scenarios someone can set these properties?